### ISSUE TITLE
[Bug]: failed to debug or execute PL object

### ISSUE THREAD

**Issue by yhilmare** [oceanbase/odc#542]
### ODC version

4.2.1

### OB version

not related

### What happened?

I create a pl object, such as procedure, which only contains out parameters. 

eg.
```SQL
create or replace procedure pl_debug_proc3(p1 out integer)
as
begin
p1 := 1234;
end;
```

I try to execute it and an exception is thrown:

<img width="647" alt="截屏2023-10-18 11 27 33" src="https://github.com/oceanbase/odc/assets/22992296/828417ee-5fee-4ee8-8d36-3470b72d3118">

by the way, this pl can be executed successfully by anonymous block.
<img width="602" alt="截屏2023-10-18 11 29 06" src="https://github.com/oceanbase/odc/assets/22992296/9fe78188-503a-4e06-b81d-a4caccccf40d">

### What did you expect to happen?

execute this pl successfully.

### How can we reproduce it (as minimally and precisely as possible)?

follow 'what happened'

**Comment by yhilmare** [comment]
the front end gives an error request which leaks `anonymousBlock`

**Comment by yhilmare** [comment]
further more, failed to execute a pl without any parameters:
<img width="775" alt="截屏2023-10-18 11 44 42" src="https://github.com/oceanbase/odc/assets/22992296/35f884f7-14fd-4ded-a88a-d398e466f38b">

**Comment by yhilmare** [comment]
which is normal on odc 4.2.0

**Comment by HSunboy** [comment]
Fixed In Commit [91dabffcdc65bf8a9c9091c411337e61977ce526](https://github.com/oceanbase/odc-client/commit/91dabffcdc65bf8a9c9091c411337e61977ce526)
            ------
            晓康
            Wed Oct 18 11:48:04 2023 +0800

**Comment by yhilmare** [comment]
> Fixed In Commit [91dabffcdc65bf8a9c9091c411337e61977ce526](https://github.com/oceanbase/odc-client/commit/91dabffcdc65bf8a9c9091c411337e61977ce526) ------ 晓康 Wed Oct 18 11:48:04 2023 +0800

😂 404 not found

**Comment by sl01388797** [comment]
![Image](https://github.com/oceanbase/odc/assets/143492234/b857a852-efb4-4e27-8545-2ad88b99aa77)

pass
